### PR TITLE
css task

### PR DESCRIPTION
**Pull request by Epresh**
Task: < css >

i commited the home page, sign-up page, login page and forget password page. and i also added the style sheet of each page.

### CODE REVIEW

**Comment by adaugochi** [comment]
@Epresh `.a, .b, .c ....` are not good class names. Please try to make use of a good class name

File: submissions/PreciousAisosaErhabor/CSS/forget-password.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+         <img src="pagent.jpeg" width=32px height="32px">PaGent</img>
+     </h1>
+     <main>
+         <h2>
+             <img src="key.jpeg" width=15px height="15px">Forgot your password</img>
+            </h2>
+            <p class="instuctions">
+                Enter your email address to change password on pagent.ng. We will send an email with instructions on how to recover your password.
+            </p>
+            <input type="text" name="email address" class="email" placeholder="Enter email address">
```
Comment (Gbensky) [inline submissions/PreciousAisosaErhabor/CSS/forget-password.html]: @Epresh Please remove the space in the value of your name attribute.

File: submissions/PreciousAisosaErhabor/CSS/registration.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+     
+     <main>
+         <label class="reg">
+             <p>
+                 Register New user
+             </p>
+            </label>
+            
+            <section>
+                <input type="text" name="firstname and othername" class="fill" placeholder="Firstname and Othername">
```
Comment (Gbensky) [inline submissions/PreciousAisosaErhabor/CSS/registration.html]: and here @Epresh

File: submissions/PreciousAisosaErhabor/CSS/registration.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                <input type="text" name="surname" class="fill" placeholder="Surname">
+                <input type="text" name="email" class="fill" placeholder="Email">
+                <input type="password" name="password" class="fill" placeholder="Password">
+                <input type="password" name="password" class="fill" placeholder="Repeat Password">
+                <input type="digit" name="phone number" class="fill" placeholder="Phone number">
```
Comment (Gbensky) [inline submissions/PreciousAisosaErhabor/CSS/registration.html]: and here @Epresh

File: submissions/PreciousAisosaErhabor/CSS/registration.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+                <input type="text" name="state of residence" class="fill" placeholder="state of residence">
```
Comment (Gbensky) [inline submissions/PreciousAisosaErhabor/CSS/registration.html]: and here @Epresh

File: submissions/PreciousAisosaErhabor/CSS/login.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+         <title>Sign-up</title>
+ </head>
+ <body>
+         <h1>
+                 <img src="pagent.jpeg" width=31px height="31px">PaGent</img>
+         </h1>
+         
+         <main  action="home-page.html">
+                 <input type="text" name="email address or member id" class="email" placeholder="Email address or Member ID">
```
Comment (Gbensky) [inline submissions/PreciousAisosaErhabor/CSS/login.html]: and here @Epresh

**Review by adaugochi — DISMISSED** [review]
well done @Epresh

File: submissions/PreciousAisosaErhabor/CSS/forget-password.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+         <h2>
+             <img src="key.jpeg" width=15px height="15px">Forgot your password</img>
+            </h2>
+            <p class="instuctions">
+                Enter your email address to change password on pagent.ng. We will send an email with instructions on how to recover your password.
+            </p>
+            <input type="text" name="email_address" class="email" placeholder="Enter email address">
+            
+            <section class="submit">
+                <button class=button type="submit" target="blank">Continue</button>
```
Comment (Gbensky) [inline submissions/PreciousAisosaErhabor/CSS/forget-password.html]: @Epresh Please add quotes to your class value. "button"

File: submissions/PreciousAisosaErhabor/CSS/forget-password.html [hunk 7]
```diff
@@ -0,0 +1,35 @@
+ <!DOCTYPE html>
+ <html lang="en">
+ <head> 
+
+    <meta charset="UTF-8">
+    <meta name="viewpoint" content="width=device-width initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+<link href = "forget-password.css" rel = "stylesheet">
```
Comment (Gbensky) [inline submissions/PreciousAisosaErhabor/CSS/forget-password.html]: @Epresh Format this properly

File: submissions/PreciousAisosaErhabor/CSS/forget-password.html [hunk 8]
```diff
@@ -0,0 +1,35 @@
+ <!DOCTYPE html>
+ <html lang="en">
+ <head> 
+
+    <meta charset="UTF-8">
+    <meta name="viewpoint" content="width=device-width initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+<link href = "forget-password.css" rel = "stylesheet"> 
+<title>https://PaGent.ng/forget-password</title>
```
Comment (Gbensky) [inline submissions/PreciousAisosaErhabor/CSS/forget-password.html:9]: @Epresh and here too

**Review by adaugochi — APPROVED** [review]

**Review by rasheedsulayman — APPROVED** [review]

**Pull request merged**